packages/client/components/DiscussionThreadInput.tsx

```diff
@@ -22,7 +22,7 @@ import CommentEditor from './TaskEditor/CommentEditor'
 import {ReplyMention, SetReplyMention} from './ThreadedItem'
 
 const Wrapper = styled('div')<{isReply: boolean; isDisabled: boolean}>(({isDisabled, isReply}) => ({
-  alignItems: 'center',
+  alignItems: 'flex-end',
```

Comment:
This keeps the avatar and the send button pinned to the bottom, instead of dynamically floating in the middle, when the input grows while adding a long comment